javascript/modules/client/src/internal/interfaces/client.ts

```diff
@@ -25,34 +27,45 @@ export interface IClient extends IClientCore {
     /** Deposits ether or an ERC20 token */
     deposit: (params: IDepositParams) => AsyncGenerator<DaoDepositStepValue>;
     /** Retrieves metadata for DAO with given identifier (address or ens domain)*/
-    getMetadata: (daoAddressOrEns: string) => Promise<DaoMetadata>;
-    /** Retrieves list of created DAOs and the corresponding metadata*/
-    getMetadataMany: (options?: DaoQueryOptions) => Promise<DaoMetadata[]>;
+    getDao: (daoAddressOrEns: string) => Promise<Dao>;
+    /** Retrieves metadata for many daos */
+    getDaos: (params: IDaoQueryParams) => Promise<DaoListItem[]>;
+  };
+  encoding: {
+    /** Computes the withdraw action payload */
+    withdrawAction: (daoAddresOrEns: string, params: IWithdrawParams) => Promise<DaoAction>;
   };
   estimation: {
     create: (params: ICreateParams) => Promise<GasFeeEstimation>;
     deposit: (params: IDepositParams) => Promise<GasFeeEstimation>;
   };
 }
 
-export interface ISearchParams{
-  pageSize?: number
-  page?: number
-}
-
 // DAO CREATION
 
 /** Holds the parameters that the DAO will be created with */
 export interface ICreateParams {
-  daoConfig: DaoConfig;
-  gsnForwarder?: string;
+  metadata: IDaoMetadata
+  ensSubdomain: string
+  plugins: IPluginInstallEntry[]
```

Comment:
There is already an interface called `PluginListItem`, I think it will be confusing calling both the same. Since the first one will be used only for installing plugins and the second one for listing them. I will rename `IPluginInstallEntry` to `IPluginInstallListItem` for consistency.

Reply:
After renaming the other interface, this will be renamed to `IPluginListItem`